test/parallel/test-async-wrap-pop-id-during-load.js

```diff
@@ -7,15 +7,17 @@ if (process.argv[2] === 'async') {
     fn();
     throw new Error();
   }
-  (async function() { await fn(); })();
-  // While the above should error, just in case it doesn't the script shouldn't
-  // fork itself indefinitely so return early.
-  return;
+  return (async function() { await fn(); })();
 }
 
 const assert = require('assert');
 const { spawnSync } = require('child_process');
 
-const ret = spawnSync(process.execPath, [__filename, 'async']);
+const ret = spawnSync(
+  process.execPath,
+  ['--stack_size=50', __filename, 'async']
```

Comment:
Aren’t we never supposed to change the stack size?

Reply:
Since this _lowers_ the size that V8 assumes to have, it should be fine. This is also the only main reason for this test to run ~3x faster. I personally think it is fine to use here because it would still work even if the flag would be a no-op. @hashseed @bmeurer are you fine with me using this in this case?